audio/pcaudiolib/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                pcaudiolib
+github.setup        espeak-ng pcaudiolib 1.1
+version             1.1
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         Portable C Audio Library
+
+long_description    Portable C Audio Library
+
+homepage            https://github.com/espeak-ng/pcaudiolib
+distname            ${version}
+checksums           rmd160  f68876d3346734e993b9d5ebfc3ef48de9161851 \
+                    sha256  449d69c3fd4ed0136d357fd42a73accb276da9b35a19258a379e3ac7ac88c148 \
+                    size    34082
+
+depends_build       port:automake \
+                    port:autoconf \
+                    port:libtool \
+                    port:pkgconfig
+
+configure.cmd        ./autogen.sh && ./configure
+build.args-append    prefix=${prefix}
+destroot.args        {*}${build.args}
```

Comment:
Is specifying prefix really necessary here? The build system should be remembering the prefix that was specified at configure time. If the build system is broken and is failing to do that, please make sure there's an upstream bug report about that and include the URL in a comment in the portfile.